server/migrations/20210729160800-offchain-topic-new-featured-fields.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.addColumn('OffchainTopics', 'featured_in_sidebar', {
+        type: Sequelize.BOOLEAN,
+        allowNull: true,
```

Comment:
this should be `false`. If we're setting the default values to `false`, then we should assume it's either featured or not, but never Null